barterdude/__init__.py

```diff
@@ -90,3 +90,18 @@ def decorator(f):
 
     def run(self):
         self.__app.run()
+
+    def __getitem__(self, key):
```

Comment:
Is it like dict inheritance?

Reply:
Yes. Looking back, I was thinking if would not be better to have `BarterDude` inherit from the `asyncworker`  `App`, since these property are already implement there, and also in the `aiohttp` server, and tested in both. What do you think?

Reply:
Since this change would mean changing a lot of code, for now I solved it just by making the `BarterDude` inherit from `MutableMapping`.

This also solved the coverage problem.

Reply:
We started with this approach, but we can't control changes at `asyncworker` and don't want to break our API. `asyncworker` is a framework and `BarterDude` is some kind of engine/template on top of `asyncworker`